content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -0,0 +1,89 @@
+==========
+AvaTax use
+==========
+
+AvaTax is a tax calculation software that can be integrated with Odoo in the United States and
+Canada. Once the :doc:`integration setup <../avatax>` is complete, the calculated tax is simple and
+automatic. This document covers the use of AvaTax in Odoo.
+
+Tax calculation
+===============
+
+Automatically calculate taxes on Odoo quotations and invoices with Avatax by confirming the
+documents. Alternatively, calculate the taxes manually by clicking the :guilabel:`Compute taxes
+using Avatax` button while these documents are in draft mode.
+
+.. tip::
+   Clicking the :guilabel:`Compute taxes using Avatax`button recalculates taxes if any product lines
+   are edited on the invoice.
+
+.. image:: avatax_use/calculate-avatax.png
+   :align: center
+   :alt: Sales quotation with the confirm and compute taxes using AvaTax buttons highlighted.
+
+The tax calculation is automatically triggered during the following circumstances:
+
+Auto Triggers
+-------------
+
+- When the sales rep sends the quote by email with :guilabel:`Send by email` button (pop-up).
+- When the customer views the online quote on the portal.
+- When a quote is confirmed and becomes a sales order.
+- When the customer views the invoice on the portal.
+- When a draft invoice is validated.
+- When the customer views the subscription in the portal.
+- When a subscription generates an invoice.
+- When the customer gets to the last screen of the eCommerce checkout.
+
+Manual Triggers
+---------------
+
+- :guilabel:`Compute taxes using Avatax` button at the bottom of the quote.
+- :guilabel:`Compute taxes using Avatax` button at the top of the invoice.
+
+.. tip::
+   Use the :guilabel:`Avalara Partner Code` field that is available on customer records, quotations,
+   and invoices to cross-reference data in Odoo and Avatax. This field is located under the
+   :menuselection:`Other info` tab of the sales order or quotation in the :guilabel:`Sales` section.
+
+   On the customer record, navigate to *Contacts app* and select a contact. Then open the
+   :guilabel:`Sales & Purchase` tab and the :guilabel:`Avalara Partner Code` under the
+   :guilabel:`Sales` section.
+
+.. important::
+   On other documents, such as subscriptions, there are also automatic and manual triggers for the
+   AvaTax calculation. The :guilabel:`Automatic Tax Mapping (AvaTax)` fiscal position is also
+   applied on those Odoo documents.
```

Comment:
The :guilabel:`Automatic Tax Mapping (AvaTax)` fiscal position is also applied on other records, like subscriptions